addons/mail/static/src/widgets/form_renderer/form_renderer_tests.js

```diff
@@ -448,6 +457,204 @@ QUnit.test('chatter should become enabled when creation done', async function (a
     );
 });
 
+QUnit.test('Form view not scrolled when switching record', async function (assert) {
+    assert.expect(7);
+
+    this.data['res.partner'].records = [{
+        activity_ids: [],
+        id: 2,
+        display_name: "second partner",
+        description: [...Array(60).keys()].join('\n'),
```

Comment:
Would add some pseudo-meaningful content than just newlines, just because this is not really realistic (this field could be trimmed in practice)

Reply:
well, this is just to have non zero scroll values. text fields are not trimmed

Reply:
You know that text fields and not trimmed, and this looks non-obvious, so more like technical detail IMO.

Having some text would make more sense from a functional PoV, and our desire is to have well-written tests, so I would still make this change.

Reply:
Still not done comment

Reply:
well, the join applies on the keys, so the indexes of the list's elements. There is text in the field....

Reply:
I consider it solved